lib/Runtime/PlatformAgnostic/Intl.h

```diff
@@ -5,14 +5,56 @@
 
 #pragma once
 
-#ifdef INTL_ICU
-
 #include "IPlatformAgnosticResource.h"
 
 namespace PlatformAgnostic
 {
 namespace Intl
 {
+    // NOTE(jahorto): Keep these enums in sync with those by the same name in Intl.js
+    // These enums are used by both WinGlob- and ICU-backed Intl
+    enum class NumberFormatStyle
+    {
+        Decimal, // Intl.NumberFormat(locale, { style: "decimal" }); // aka in our code as "number"
```

Comment:
We should probably add `Min = 0, <next value> = Min` to all of these enums so we don't have to cast 0 as a minimum case when checking.